product-docs/integrations/service-accounts.md

```
@@ -10,9 +10,22 @@ A **service account** is an identity for a script, CI pipeline, or external syst
 This is a **PRO** feature. It is gated by the `Service accounts` feature flag (see [feature-flags.md](../configuration/settings/feature-flags.md "mention")). While the flag is off, the **Service accounts** menu stays hidden and its API is unreachable.
 {% endhint %}
 
+### Identity source: local or federated
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)